benchmarks/reduce.py

```diff
@@ -5,6 +5,7 @@
 import arkouda as ak
 
 OPS = ('sum', 'prod', 'min', 'max')
+TYPES = ('int64', 'float64')
```

Comment:
`bool` reductions are also supported with `sum`, `prod`, `min`, and `max`

Reply:
As-is the benchmark doesn't work with bools (there's no case to handle creating the arrays of bools). I can add that here too if you want, or I can do that in a follow up PR.